apps/explorer/src/ui/stories/SenderRecipientAddress.stories.tsx

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { type Meta, type StoryObj } from '@storybook/react';
+import { MemoryRouter } from 'react-router-dom';
+
+import {
+    SenderRecipientAddress,
+    type SenderRecipientAddressProps,
+} from '../SenderRecipientAddress';
+
+export default {
+    component: SenderRecipientAddress,
+    decorators: [
+        (Story) => (
+            <MemoryRouter>
+                <Story />
+            </MemoryRouter>
+        ),
+    ],
+} as Meta;
+
+export const TxnAddress: StoryObj<SenderRecipientAddressProps> = {
+    args: {
+        address: '0x813f1adee5abb1e00dfa653bb827856106e56764',
+        isSender: true,
```

Comment:
Missing a story for `isSender: false`

Reply:
The Storybooks controls have this.

Reply:
Yes but we shouldn’t depend on storybook controls. The stories themselves should be exhaustive.